### ISSUE TITLE
Please provide a server client example that uses secure WebSockets

### ISSUE THREAD

**Issue by skoehler** [shamblett/mqtt_client#242]
In the examples folder, there does not seem to be an example that uses the server client to connect via  secure websocket connection.

In my case, the websocket URL would be `wss://hostname/path/to/ws`. Looking at the documentation, I came up with
```
client = MqttServerClient('host', '');
client.port = 443;
client.secure = true;
client.useWebSocket = true;
```
but I wouldn't know how to set the path part of URL, in this case that is `/path/to/ws`. Without the path, things won't work.

A good example would come in handy.

Edit: `client.secure` is only for plain TCP connections, not for websockets. So an additional question is how to enable TLS.

**Comment by skoehler** [comment]
I have been digging around in the sources. Turns out, that the hostname string is eventually handed down to the MqttServerWsConnection class, which treats it as a URL.

So things might be as simple as
```
client = MqttServerClient('wss://hostname/path/to/ws', '');
client.useWebSocket = true;
```

**Comment by shamblett** [comment]
OK. I'll update the documentation

**Comment by shamblett** [comment]
Updated and server client websocket examples added, package re-released at version 9.0.0
